data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -156,5 +187,30 @@
             <column name="tenant_id"/>
             <column name="alias"/>
         </createIndex>
+        <createTable tableName="crypto_signing_key_material">
+            <column name="wrapping_key_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="signing_key_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="signing_key_material" type="VARBINARY(1048576)">
+                <constraints nullable="false"/>
+            </column>
+            <!-- when the key material was generated -->
+            <column name="created" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+
+            <!-- when the key material was generated -->
+            <column name="accessed" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP">
```

Comment:
comment says `generated` but column says `accessed`. As above, what is the column for?

Reply:
removed